api-reference/beta/toc.yml

```
@@ -11513,6 +11513,13 @@ items:
               href: api/unifiedRoleManagementPolicyRule-get.md
             - name: Update
               href: api/unifiedRoleManagementPolicyRule-update.md
+        - name: Tenant information
+          href: resources/tenantrelationships.md
+          items:
+          - name: Find by domain name
+            href: api/tenantrelationships-findtenantinformationbydomainname.md
+          - name: Find by tenant ID
+            href: api/tenantrelationships-findtenantinformationbytenantid.md
```

To reflect the file name changes
```suggestion
          href: resources/tenantrelationshiproot.md
          items:
          - name: Find by domain name
            href: api/tenantrelationshiproot-findtenantinformationbydomainname.md
          - name: Find by tenant ID
            href: api/tenantrelationshiproot-findtenantinformationbytenantid.md
```